types/expo/expo-tests.tsx

```diff
@@ -8,40 +8,39 @@ import {
     AdMobInterstitial,
     AdMobRewarded,
     Amplitude,
+    AppLoading,
     Asset,
-    AuthSession,
     Audio,
-    AppLoading,
+    AuthSession,
     BarCodeScanner,
-    BlurViewProps,
     BlurView,
     Brightness,
+    Calendar,
     Camera,
     CameraObject,
+    Constants,
     DocumentPicker,
     Facebook,
     FacebookAds,
+    FaceDetector,
     FileSystem,
-    ImagePicker,
+    Haptic,
     ImageManipulator,
-    FaceDetector,
-    Linking,
-    Svg,
+    ImagePicker,
     IntentLauncherAndroid,
     KeepAwake,
     LinearGradient,
+    Linking,
+    Location,
+    MailComposer,
+    MapView,
+    MediaLibrary,
     Permissions,
     PublisherBanner,
     registerRootComponent,
     ScreenOrientation,
-    SQLite,
-    Calendar,
-    MailComposer,
-    Location,
-    Updates,
-    MediaLibrary,
-    Haptic,
-    Constants
+    Svg,
+    Updates
 } from 'expo';
 
```

Comment:
```suggestion
import { Region, EdgePadding, EdgeInsets, MapEvent, KmlMapEvent, EventUserLocation, MapStyleElement } from './mapview';
```

Reply:
Looks good. Thank you so much for the effort.